ideascube/serveradmin/catalog.py

```diff
@@ -154,8 +154,8 @@ def commit(self):
         for libpath in glob(os.path.join(libdir, '*.xml')):
             zimname = os.path.basename(libpath)[:-4]
 
-            with open(libpath, 'r') as f:
-                et = etree.parse(f)
+            with open(libpath, mode='r', encoding='utf-8') as f:
+                et = etree.fromstring(f.read())
```

Comment:
Hum, you're right, I've got in now.

I'll see on kiwix side to add the utf8 encoding.